_SQLServer/TableLoadAbilities.sql

```diff
@@ -1,6 +1,193 @@
+--1.0
+--Angry Bear
+insert into Abilities (Name, AbilityResource, AbilityType, AbilityCost, AbilityAttribute, AbilityRange, AbilityDescription, KeywordVersion) VALUES ('Bear Charge', 'Action', 'Offense', 2, 0, 1, '+1R Attack, Knockdown', '1.0');
+insert into Abilities (Name, AbilityResource, AbilityType, AbilityCost, AbilityAttribute, AbilityRange, AbilityDescription, KeywordVersion) VALUES ('Bear Hug', 'Action', 'Offense', 2, 0, 1, 'Immobile, Weak', '1.0');
+insert into Abilities (Name, AbilityResource, AbilityType, AbilityCost, AbilityAttribute, AbilityRange, AbilityDescription, KeywordVersion) VALUES ('Shapeshift', 'Action', 'Support', 1, -1, -1, 'Deeproot Druid', '1.0');
+insert into Abilities (Name, AbilityResource, AbilityType, AbilityCost, AbilityAttribute, AbilityRange, AbilityDescription, KeywordVersion) VALUES ('Nourishing Berries', 'Potion', 'Support', 1, -1, -1, 'All Heroes gain Healer', '1.0');
 
 
+--Calico Kate
+insert into Abilities (Name, AbilityResource, AbilityType, AbilityCost, AbilityAttribute, AbilityRange, AbilityDescription, KeywordVersion) VALUES ('Bomb''s Away', 'Action', 'Offense', 2, -1, -1, 'Range 6, Burst 1, Knockdown, Dangerous', '1.0');--? Check values
+insert into Abilities (Name, AbilityResource, AbilityType, AbilityCost, AbilityAttribute, AbilityRange, AbilityDescription, KeywordVersion) VALUES ('Pirate Snatch', 'Action', 'Offense', 2, 1, -1, '+1R Attack, If Pirate Snatch inflicts a wound advance the Loot-O-Meter one additional space', '1.0');--? Check values
+insert into Abilities (Name, AbilityResource, AbilityType, AbilityCost, AbilityAttribute, AbilityRange, AbilityDescription, KeywordVersion) VALUES ('Skully', 'Potion', 'Offense', 1, 1, -1, 'R Wave 2, +1R Attack, Dangerous', '1.0');--? Check values
 
+
+--Candy & Cola
+insert into Abilities (Name, AbilityResource, AbilityType, AbilityCost, AbilityAttribute, AbilityRange, AbilityDescription, KeywordVersion) VALUES ('Vending Machine', 'Action', 'Support', 2, -1, 6, 'Augment, Range 6. Remove a single potion counter from Candy''s card and place it on target Hero''s card.', '1.0');--? Check values
+insert into Abilities (Name, AbilityResource, AbilityType, AbilityCost, AbilityAttribute, AbilityRange, AbilityDescription, KeywordVersion) VALUES ('Soothing Soda', 'Potion', 'Support', -1, 1, -1, 'Immune: Status Effects.', '1.0');--? Check values
+insert into Abilities (Name, AbilityResource, AbilityType, AbilityCost, AbilityAttribute, AbilityRange, AbilityDescription, KeywordVersion) VALUES ('Sugar Rush', 'Potion', 'Support', 3, 1, -1, '+2ST', '1.0');--? Check values
+
+
+--Celestial Herald
+insert into Abilities (Name, AbilityResource, AbilityType, AbilityCost, AbilityAttribute, AbilityRange, AbilityDescription, KeywordVersion) VALUES ('Arrow Storm', 'Action', 'Offense', 2, -1, 4, 'DEX vs ARM, Spray 4', '1.0');--? Check values
```

Comment:
AbilityRange should probably be -1.
This way the attribute and range can be ignored on the webpage.

Reply:
AbilityAttribute should probably be 4.